lib/ok.ex

```diff
@@ -161,22 +161,39 @@ defmodule OK do
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
       ...>   b <- safe_div(a, 2)
+      ...>   OK.success a + b
+      ...> end
+      {:ok, 6.0}
+
+      iex> OK.with do
+      ...>   a <- safe_div(8, 2)
+      ...>   b <- safe_div(a, 2)
       ...>   {:ok, a + b}
       ...> end
       {:ok, 6.0}
 
       iex> OK.with do
       ...>   a <- safe_div(8, 2)
+      ...>   b = 2.0
+      ...>   OK.success a + b
```

Comment:
That, was a genuine question from me. Not sure of the answer at all. If Elixir 1.4 requires parens to always be used for functions but not macros then I am curious why that distinction exists. Maybe its a forum question for me